code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -84,6 +84,9 @@
 	if (is_type_in_list(target, can_be_placed_into))
 		return
 
+	if(ishuman(target)) //Splashing handled in attack now
```

Comment:
Good question; with borgs they have no mouths, but if you wanted to feed a monkey I guess this would have to be iscarbon(target)?

Aliens too, in that regard